README.md

```diff
@@ -43,6 +43,36 @@ Covered in the [process document](docs/process.md).
 ### Testing
 The **Wikipedia** scheme is configured to execute the project's iOS unit tests, which can be run using the `Cmd+U` hotkey or the **Product->Test** menu bar action. Screenshot tests will fail unless you are running on one of the configurations defined by `configurations_to_test_on_pull` in `fastlane/Fastfile`.
 
+### Targets
+#### Wikipedia
+Points to production servers.
+#### Staging and Wikipedia (apps staging environment)
```

Comment:
Rename to just Staging to match scheme name?

Reply:
Actually nevermind, I thought it was referencing one scheme but I see now that it's both.

Reply:
It seems like now we can get rid of `Wikipedia (apps staging environment)` since it's close enough to `Staging`, so I removed it.